config/web.ini.sample

```diff
@@ -15,4 +15,4 @@
 # addr = 127.0.0.1
 
 # Port to listen on.
-port = 80
+port = 8080
```

Comment:
If you want to change this then you should also update the README. Ideally it would be in its own PR.